client/src/components/Nav.js

```diff
@@ -138,7 +138,7 @@ class BaseNav extends Component {
 
 				<SidebarLink linkTo="/help" handleOnClick={resetPages}>Help</SidebarLink>
 
-				{(currentUser.isAdmin() || currentUser.isSuperUser()) &&
+				{(currentUser.isAdmin()) &&
```

Comment:
I'd prefer:
```{(currentUser.isAdmin() /* FIXME: enable this again when the problem is fixed: || currentUser.isSuperUser()*/) &&```
so it at least shows up in the task list of one's IDE.

Reply:
Sure (although when fixed we will expose it to all users)